packages/ia-components/sandbox/theatres/components/track-list/track-list.jsx

```diff
@@ -87,16 +88,20 @@ class TheatreTrackList extends Component {
     } = this.props;
 
     if (!tracks.length) return <p className="no-tracks">No tracks to display.</p>;
+
+    const [ firstTrack = {} ] = tracks;
+    const trackNumberToHighlight = selectedTrack || firstTrack.trackNumber || null;
```

Comment:
QUESTION: Does the final `|| null` serve a useful purpose?

Reply:
another defensive reflex.